apps/omg/test/omg/ethereum_client_monitor_test.exs

```diff
@@ -111,46 +121,31 @@ defmodule OMG.EthereumClientMonitorTest do
     pid = Process.whereis(EthereumClientMonitor)
     true = is_pid(pid)
 
-    _ =
-      pull_client_alarm(100, [
-        %{
-          details: %{node: Node.self(), reporter: EthereumClientMonitor},
-          id: :ethereum_client_connection
-        }
-      ])
+    node = Node.self()
+
+    :ok =
+      call_while(
```

Comment:
same here, this is an unnecessary rewrite (does not address any issues) and should be reverted